src/inferred_mode/inferred_mode.ts

```diff
@@ -623,8 +595,7 @@ export default class InferredQueryHandler {
     response.message.results = Object.values(combinedResponse.message.results).sort((a, b) => {
       return b.analyses[0].score - a.analyses[0].score ? b.analyses[0].score - a.analyses[0].score : 0;
     });
-    // trim extra results and prune kg
-    response.message.results = response.message.results.slice(0, this.CREATIVE_LIMIT);
```

Comment:
Results should still be truncated to the top 500 at the end per Translator expectations.